app/helpers/application_helper.rb

```diff
@@ -385,6 +385,12 @@ def show_parent?(obj)
     (obj.new_record? && obj.class.count > 0) || (!obj.new_record? && obj.class.count > 1)
   end
 
+  def documentation_button(section)
+    link_to(icon_text('question-sign', _('Documentation'), :class => 'icon-white'),
+            "http://www.theforeman.org/manuals/#{SETTINGS[:version].short}/index.html##{section}",
```

Comment:
will `SETTINGS[:version].short` work with nightly? we don't have 1.9 manual until 1.9 is branched I think